Formula/adios2.rb

```diff
@@ -24,13 +23,6 @@ class Adios2 < Formula
   depends_on "zeromq"
   uses_from_macos "bzip2"
 
```

Comment:
```suggestion

  # macOS 10.13 configuration-time issue detecting float types
  # reference: https://github.com/ornladios/ADIOS2/pull/2305
  patch do
    url "https://github.com/ornladios/ADIOS2/pull/2305.patch?full_index=1"
    sha256 "6d0b84af71d6ccf4cf1cdad5e064cca837d505334316e7e78d18fa30a959666a"
  end
```

Reply:
+1

Let's wait for ornladios/ADIOS2#2305 to get merged before adding this.